src/EventPayload.ts

```diff
@@ -71,22 +80,48 @@ export interface EventPayload {
   internalUser?: boolean;
   /** The IP address for the event.*/
   ip?: {
-    /** The IPv4 address associated with the event. */
+    /** The IPv4 address associated with the event.
+     *
+     * Defaults to '$auto' which indicates to use the Source IP address from the IP header of the request.
+     * This IP address is only used if the `Yext-Opt-In: ip` header is present."
```

Comment:
This IP address is only used if the `Yext-Opt-In: ip` header is present. 

Is this actually true? I didn't think it was.

Reply:
I copied this from the Events Schema description, so we'd want to update that too if this isn't the case.

Reply:
Updated the schema in alpha.